components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/policy/finder/PolicyFinderModule.java

```diff
@@ -140,18 +140,5 @@ public Map<String, Set<AttributeDTO>> getSearchAttributes(String identifier,
      */
     public boolean isDefaultCategoriesSupported();
 
-    /**
-     * returns whether this module supports for policy ordering.
-     *
-     * @return whether supported or not
-     */
-    public boolean isPolicyOrderingSupport();
```

Comment:
Have to check whether removing this is intentional.

Reply:
Might have been removed since all usages of `isPolicyOrderingSupport()` evaluate to true. To minimize impact, better to keep it as it is.